modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * .adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual machine overview* tab displays information about the *Operating system*, *Time zone*, and *Logged in users*.
+
+The *Virtual machine overview* tab shows information about the guest operating system that was retrieved and displayed by the QEMU guest agent. If the QEMU guest agent is not installed, the *Virtual Machine Overview* tab displays information about the operating system that was specified when the virtual machine was created.
```

Comment:
> about the guest operating system that was retrieved and displayed by the QEMU guest agent.

This language doesn't sound quite right. I would use something like "about the guest operating system installed on the virtual machine."

Reply:
Updated the content.